src/components/MenuButton/MenuButton.tsx

```diff
@@ -144,4 +145,31 @@ export default class MenuButton extends React.Component<MenuButtonProps> {
   private getMenuItemContent(props: IContextualMenuItemProps) {
     return <MenuButtonItem {...props} />;
   }
+
+  private getStyles = (): IButtonStyles => {
+    return {
+      root: {
+        border: 0,
+        padding: 0,
+        opacity: 0.7,
+        width: `${this.props.iconSize}px`,
+        height: `${this.props.iconSize}px`,
+        color: '#495361',
```

Comment:
It may be time to create a TS file with all these colors, and have it live side-by-side with the CSS one. Having these colors sprinkled in TS code doesn't look great.

Reply:
i agree, i don't know what color that is just looking at the PR :(